docs/Recommendations.md

```diff
@@ -161,6 +163,83 @@ backend static-backend
 [why-use]: https://web.archive.org/web/20190821102906/https://medium.com/intrinsic/why-should-i-use-a-reverse-proxy-if-node-js-is-production-ready-5a079408b2ca
 [haproxy]: https://www.haproxy.org/
 
+### Nginx
+
+```nginx
+upstream fastify_app {
+  # more info: http://nginx.org/en/docs/http/ngx_http_upstream_module.html
+  server 10.10.11.1:80;
+  server 10.10.11.2:80;
+  server 10.10.11.3:80 backup;
```

Comment:
I think we should add some docs about folks configuring this ip address.

Reply:
I just take it from haproxy example.

It's better to be simpler example focus on TLS offloader and load balancing technique. HAProxy example is more complicated i think.